<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainUi</class>
 <widget class="QMainWindow" name="mainUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Double Barrel Server Monitor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/icons/compedShotgun.png</normaloff>:/icons/compedShotgun.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="logSplitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTreeWidget" name="serverTree">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Script Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Status</string>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="logDock" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="logOutput"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="showLogLabel">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../icons.qrc">:/icons/monitor.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="showLogCheck">
        <property name="text">
         <string>Show log</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string extracomment="Add Server"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeButton">
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="toolTip">
         <string extracomment="Remove Server"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>893</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuManage">
    <property name="title">
     <string>Manage</string>
    </property>
    <addaction name="addServerAction"/>
    <addaction name="removeServerAction"/>
    <addaction name="separator"/>
    <addaction name="restartServerAction"/>
    <addaction name="separator"/>
    <addaction name="startServerAction"/>
    <addaction name="stopServerAction"/>
   </widget>
   <widget class="QMenu" name="menuLog">
    <property name="title">
     <string>Log</string>
    </property>
    <addaction name="showLogAction"/>
    <addaction name="hideLogAction"/>
    <addaction name="separator"/>
    <addaction name="verticalSplitAction"/>
    <addaction name="horizontalSplitAction"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Display</string>
    </property>
    <addaction name="simpleModeAction"/>
    <addaction name="advancedModeAction"/>
   </widget>
   <addaction name="menuManage"/>
   <addaction name="menuLog"/>
   <addaction name="menuPreferences"/>
  </widget>
  <action name="verticalSplitAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Vertical Split</string>
   </property>
  </action>
  <action name="horizontalSplitAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Horizontal Split</string>
   </property>
  </action>
  <action name="showLogAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Log</string>
   </property>
  </action>
  <action name="addServerAction">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/plus.png</normaloff>:/icons/plus.png</iconset>
   </property>
   <property name="text">
    <string>Add Server</string>
   </property>
  </action>
  <action name="removeServerAction">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/minus.png</normaloff>:/icons/minus.png</iconset>
   </property>
   <property name="text">
    <string>Remove Server</string>
   </property>
  </action>
  <action name="hideLogAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide Log</string>
   </property>
  </action>
  <action name="actionUnlock">
   <property name="text">
    <string>Unlock</string>
   </property>
  </action>
  <action name="restartServerAction">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Restart Server</string>
   </property>
  </action>
  <action name="simpleModeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/star.png</normaloff>:/icons/star.png</iconset>
   </property>
   <property name="text">
    <string>Simple</string>
   </property>
  </action>
  <action name="advancedModeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/advanced.png</normaloff>:/icons/advanced.png</iconset>
   </property>
   <property name="text">
    <string>Advanced</string>
   </property>
  </action>
  <action name="setServerFileAction">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/server.png</normaloff>:/icons/server.png</iconset>
   </property>
   <property name="text">
    <string>Set Server File</string>
   </property>
  </action>
  <action name="startServerAction">
   <property name="text">
    <string>Start Server</string>
   </property>
  </action>
  <action name="stopServerAction">
   <property name="text">
    <string>Stop Server</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
  <include location="../icons.qrc"/>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
